tests/integration/scenario.go

```diff
@@ -29,4 +29,5 @@ func initScenario(ctx *godog.ScenarioContext) {
 	ctx.Step(`^Application "([^"]*)" in namespace "([^"]*)" has proxy with "([^"]*)" set to cpu - "([^"]*)" and memory - "([^"]*)"$`, steps.ApplicationHasProxyResourcesSetToCpuAndMemory)
 	ctx.Step(`^Application pod "([^"]*)" in namespace "([^"]*)" has Istio proxy "([^"]*)"$`, steps.ApplicationPodShouldHaveIstioProxy)
 	ctx.Step(`^Destination rule "([^"]*)" in namespace "([^"]*)" with host "([^"]*)" exists$`, steps.CreateDestinationRule)
+	ctx.Step(`^And Istio is manually uninstalled$`, steps.UninstallIstio)
```

Comment:
The test steps are skipped since this step couldn't be found. Unfortunately, this doesn't cause an exit code unequal to 0 for the test run, yet. I'll push changes to fix this behaviour.
```suggestion
	ctx.Step(`^Istio is manually uninstalled$`, steps.UninstallIstio)
```